Source/SVGKImage.m

```diff
@@ -222,6 +226,14 @@ + (SVGKImage*) imageWithData:(NSData *)newNSData
     }
 }
 
++ (SVGKImage*) imageWithDataAsynchronously:(NSData *)newNSData onCompletion:(SVGKImageAsynchronousLoadingDelegate)blockCompleted
+{
+	NSParameterAssert(newNSData != nil);
+	SVGKitLogWarn(@"Creating an SVG from raw data; this is not recommended: SVG requires knowledge of at least the URL where it came from (as it can contain relative file-links internally). You should use the method [SVGKImage initWithSource:] instead and specify an SVGKSource with more detail" );
+
+	return [self imageWithSource:[SVGKSourceNSData sourceFromData:newNSData] onCompletion:blockCompleted];
```

Comment:
I'm referring to the SVGKSourceNSData initialization not the imageWithSource method.